### PR TITLE
updates how we send created_at to Rogue

### PR DESCRIPTION
**Pull request by chloealee**
#### What's this PR do?
Sends `created_at` time in correct format that we want to send to Rogue. 

#### How should this be reviewed?
👀 

#### Relevant tickets
[This](https://www.pivotaltracker.com/n/projects/2019429/stories/156165836) Pivotal Card and this PR in Rogue.

### CODE REVIEW

File: app/Jobs/ImportFacebookSharePosts.php [hunk 1]
```diff
@@ -78,9 +78,8 @@ public function handle(Rogue $rogue)
                 'status' => 'accepted',
                 'source' => 'importer-client',
                 'source_details' => json_encode(['original-source' => $record['event.source']]),
-                'created_at' => strtotime($record['to_timestamp']),
+                'created_at' => date(DATE_ATOM, strtotime($record['to_timestamp'])),
```
Comment (katiecrane) [inline app/Jobs/ImportFacebookSharePosts.php]: @chloealee what's the `to_timestamp` column look like?

Comment (chloealee) [reply app/Jobs/ImportFacebookSharePosts.php]: @katiecrane here's an example: `6/8/2018 18:27:00`

Comment (chloealee) [reply app/Jobs/ImportFacebookSharePosts.php]: ohhh @katiecrane maybe I should do something like you and @sbsmith86 did in your imports:
```
strtolower(Carbon::parse($record['Started registration'])->format('F-Y'));
```

Comment (chloealee) [reply app/Jobs/ImportFacebookSharePosts.php]: oh wait, I think I am wrong I don't think we want to format that like that here.

Comment (katiecrane) [reply app/Jobs/ImportFacebookSharePosts.php]: I think you might be able to do something like this instead if you want to use Carbon:
```
Carbon::parse($record['to_timestamp'])->toDateTimeString();
```
Does that give you the right format?

Comment (chloealee) [reply app/Jobs/ImportFacebookSharePosts.php]: ah yes it does! Thanks @katiecrane ! Updated - I like that way better :)

**Pull request merged**